src/App.tsx

```diff
@@ -22,18 +22,106 @@ class App extends React.Component<{}, State> {
     ],
   };
 
-  render() {
-    const { images } = this.state;
+  const [step, setStep] = useState(3);
+  const [frameSize, setFrameSize] = useState(3);
+  const [itemWidth, setItemWidth] = useState(130);
+  const [animationDuration, setAnimationDuration] = useState(1000);
+  const [infinite, setInfinite] = useState(false);
```

Comment:
```suggestion
  const [inputs, setInputs] = useState({
    step: 3,
    frameSize: 3;
    ....
  })
```
make one state for all inputs as object. and change it in one hanler using event.target.name for change proper inputs field.